civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -20,8 +20,12 @@
       overflow: visible !important;
     }
 
-    #bootstrap-theme .dropdown-menu[datepicker-popup-wrap] {
-      width: 316px;
+    #bootstrap-theme {
+      display: inline-block;
```

Comment:
`#bootstrap-theme` it's just the namespace, i don't think we should really apply any style to it. What's the purpose of this `inline-block`? Can't you apply it to some other element?